.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!-- Have you followed the guidelines in our Contributing document?
+https://github.com/vrk-kpa/suomifi-ui-components/blob/develop/CONTRIBUTING.md
+Have you checked to ensure there aren't other open Pull Requests for the same update/change?
+Hopefully you did not remove any lock-files, tests or linter-rules to pass the CI-automation? -->
+
+## Description
+
+<!-- Describe your changes in detail -->
+<!-- Add [Feature] or [BreakingChange] to the title -->
+
+## Related Issue
+
+<!-- If suggesting a new feature or change, please discuss it in an issue first -->
+<!-- If fixing a bug, please link to the issue here: -->
+
+Closes #
+
+## Motivation and Context
+
+<!-- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+
+<!-- Please describe in detail how you tested your changes. -->
+
```

Comment:
Is this really relevant here or what is the value of the description?

Reply:
Yeah can remove that